_articles/id/starting-a-project.md

```diff
@@ -161,9 +161,9 @@ Seringkali, banyak orang menghindari untuk menulis README karena mereka merasa b
 
 Untuk insipirasi lainnya, Silahkan coba ["Membuat README lebih Terbaca"](https://pages.18f.gov/open-source-guide/making-readmes-readable/) milik @18F atau [Template README](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2) milik @PurpleBooth untuk menulis README yang lengkap.
 
-Ketika Anda menyertakan dokumen README pada direktori induk, GitHub akan secara otomatis menampilkannya pada homepage repositori.
+Ketika Anda menyertakan dokumen README pada direktori induk, GitHub akan secara otomatis menampilkannya pada halaman utama repositori.
 
-### Menulis panduan kontribusi Anda
+### Panduan menulis kontribusi
```

Comment:
This is the same change as you made in #153; apparently the original is correct https://github.com/github/opensource.guide/pull/1535#issuecomment-602004078